agir/front/components/globalContext/nonReactRoutes.config.js

```diff
@@ -0,0 +1,32 @@
+export default {
+  search: "/recherche",
+  personalInformation: "/profil/identite/",
+  contactConfiguration: "/profil/contact/",
+  createGroup: "/groupes/creer/",
+  groupsMap: "/carte/groupes/",
+  groupMapPage: "/groupes/carte/",
+  eventsMap: "/carte/evenements/",
+  eventMapPage: "/evenements/carte/",
```

Comment:
```suggestion
  groupsMap: "/carte/groupes/",
  eventsMap: "/carte/evenements/",
```
Il faudra vérifier s'il reste des endroits où on utilise encore les routes qui venaient du backend, mais ces deux routes sont des page React